.github/ISSUE_TEMPLATE/2-bug-report.md

```diff
@@ -7,20 +7,47 @@ assignees: ''
 
 ---
 
-<!---
-Please do not ask for help or general inquiry here, use RPCS3 Discord server instead.
-
-Please read Contributing guide under Helpful resources on the right. →
---->
-
 ## Quick summary
-Please summarize the issue here
+Please briefly describe what is not working correctly.
 
 ## Details
-Please provide as much detail as you can here, including
-- expected (real hw) and observed (emulator) behavior
-- full logs (not from UI)
-- back traces (for hard crashes)
-- screenshots, Renderdoc, and RSX captures (for visual issues)
-- your system configuration (OS, CPU, GPU, driver versions, etc)
-- anything else you deem to be important
+
+Please describe the problem as accurately as possible.
+
+#### 1. Please attach RPCS3 log
+* Run the game
+* Completely close RPCS3, or move to the next step in case it has crashed.
+* Locate RPCS3 log file
+	+ ```RPCS3.log``` (It can show up as just RPCS3 and have a notepad icon)
+	or
+	+ ```RPCS3.log.gz``` (It can show up as RPCS3.log and have a zip or rar icon)
+![image](https://user-images.githubusercontent.com/44116740/84433247-aa15fa80-ac36-11ea-913e-6fe25a1d040e.png)
+	On Linux it will be in ```~/.cache/rpcs3/```.
+	On Windows it will be near the executable.
+* Attach the log
+	+ Drag and drop your log file into the issue.
+	+ Or upload it to the cloud, such as Dropbox, Mega etc. 
+
+#### 2. If you describe graphical issue, please provide RSX capture and a RenderDoc capture that demonstrate the issue.
+* To create an RSX capture, use _Create_ _RSX_ _Capture_ under _Utilities_.
+Captures will be stored in RPCS3 folder → captures.
+	+ Compress your capture with 7z, Rar etc. 
+	And	drag and drop it into the issue.
+	+ Or upload it to the cloud, such as Dropbox, Mega etc.
+* To create RenderDoc capture, please refer to [RenderDoc documentation](https://renderdoc.org/docs/how/how_capture_frame.html).
```

Comment:
```suggestion
* To create a RenderDoc capture, please refer to [RenderDoc's documentation](https://renderdoc.org/docs/how/how_capture_frame.html).
```

Reply:
Everything seems fair. Will correct in a sec.